src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const input = sourceString.split(';');
+  const result = {};
+
+  input.forEach((item) => {
```

Comment:
You can use `reduce` to generate an object. `item` is not informative name as well